express-simple/README.md

```diff
@@ -1,3 +1,7 @@
-# node-dmbz1u
+# Simple express demo
 
-[Edit on StackBlitz ⚡️](https://stackblitz.com/edit/node-dmbz1u)
\ No newline at end of file
+> Express: fast, unopinionated, minimalist web framework for Node.js
+
+This project shows a simple express server serving a single HTML page and using `express.static` to serve static files.
+
+Check out the [express documentation](https://expressjs.com/) for more information.
```

Comment:
Apparently we had just saved an automatically generated `README.md` pushed to GitHub by the StackBlitz editor.